### DISCUSSION TITLE
Problem Motion Planning While Holding Objects

### DISCUSSION THREAD

**Discussion by joshmansky** [NVlabs/curobo#360] [Software Q&A]
Hello, I have been working with cuRobo for motion planning with a dual arm table top setup for a simple stacking/peg-in-hole task. The problem I am currently having is that while the arms are executing all of the correct movements when they do not grasp the blocks, when they do grasp the blocks after lifting them up they stop. They are not completely stopped as when  I led the simulation run, after about 10 minutes they make another movement(I have not let it run long enough to see if they eventually complete the full demo); however, in the same simulation, when I maunally knock the blocks out of their grasp they move and finish the motions very quickly. Thus, I was wondering if anyone knew what the problem and potential soultion is. I am very confident it is not collisions between the collision spheres in the hand and the blocks as I call the attach_object() function and set in the yaml file to ignore self-collisions between the blocks and hands, but it is possible I am still be missing something with the collisions. The video below shows my problem. 
[Screencast from 08-08-2024 09:21:31 AM.webm](https://github.com/user-attachments/assets/64abb260-53fb-43b8-b404-f17024816916)

**Comment by balakumar-s** [comment]
If start state is in collision, this will be reported by MotionGenResult.status. Can you print out what this variable contains?

My guess is that when attach_object is called, it's only attaching the object to one of the end-effectors.
